time node_timestamp
string node_label
string node_label_str
float32 color_r
float32 color_g
float32 color_b
